Fix grammar and styling in `Storyboard/Scripting/Variables` (#9262)
ppy/osu-wiki: wiki/Storyboard/Scripting/Variables/en.md

# Storyboard scripting variables

> **Variables** are abstract containers holding custom values that can be used elsewhere in an `.osb` file (note that the value cannot be changed during gameplay, so for those with programming experience, think of it more like a constant). `.osu` files *do not* support this.

Author: Tell me if I should remove the part in brackets too. I'm not sure if this is too informal.

Reviewer: ```suggestion
**Variables** are custom aliases for other values, typically long or common strings, which can be reused elsewhere in `.osb` files. These cannot be changed dynamically during gameplay, meaning they are constant values. They are *not supported* in `.osu` files.
```

## Usage

> The declaration of variables occurs in the separate \[Variables\] section located at the top of the `.osb` file:

Reviewer: ```suggestion
The declaration of variables occurs in a separate \[Variables\] section typically located at the top of the `.osb` file:
```

```
[Variables]
$colour_link=0,255,0
$sample_path="Sample.png"
```

> Variables are used in the code by typing their name (left hand side of the declaration, including the $). For instance, with the above declarations, this:

Reviewer: ```suggestion
Variables are used in the code by typing their name (left-hand side of the declaration, including the $). For instance, with the above declarations, this:
```

_C,0,6000,7000,12,12,12
```

> Saving will replace the "12"s in colour specification with `$number_of_loops`.

Reviewer: ```suggestion
Saving will replace the "12"s in the colour specification with `$number_of_loops`.
```
